plugins/contactpressuresensorsarray/src/ContactPressureSensorsArray.cc

```diff
@@ -69,12 +69,7 @@ namespace gazebo
         // Insert the pointer in the singleton handler for retrieving it in the yarp driver
         GazeboYarpPlugins::Handler::getHandler()->setRobot(boost::get_pointer(_parent));
 
-#if GAZEBO_MAJOR_VERSION >= 7
-        m_gazeboRobotName = _parent->GetScopedName();
-#else
-        m_gazeboRobotName = _parent->ScopedName();
-#endif
-    
+        m_gazeboRobotName = _parent->GetName();
```

Comment:
You went from scoped name to name.. they are the same?

Reply:
I don't know why I made this implusive decision. Reverting back to GetScopedName() since this method is supported through gazebo8 as well